dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -12,6 +12,13 @@ def load_user
     @user = current_user
   end
 
+  # GET /api/v1/users/<user_id>/school_name
+  def get_school_name
+    render json: {
+      school_name: current_user&.school
+    }
+  end
```

Comment:
This whole controller strikes me as very odd.  Our route takes a `user_id` parameter, but it _must_ be `me` or the current user's id, and our controller method here ignores the parameter anyway.  Something we'd want to clean up in the future?  At the very least, I'd suggest this method use `@user.school`, and have a test that confirms we can only get the current_user's school name, so if the `load_user` method is changed in the future we know this behavior is preserved.